mobly/suite_runner.py

```diff
@@ -13,8 +13,10 @@
 # limitations under the License.
 """Runner for Mobly test suites.
```

Comment:
I think we need to further clarify the purposes of this suite_runner and the base_suite
Both of the are meant to be boilerplate code or examples for users to construct their own suite code right?

What we provide works, but users should not feel they have to do things this way.
E.g. the argument format can be whatever users want to define.